src/Models/UptimeCheck.php

```diff
@@ -92,7 +97,7 @@ protected static function booted()
                 return;
             }
 
-            if ($uptime->getOriginal('status') != $uptime->status) {
+            if ($uptime->getOriginal('status') != $uptime->status && $uptime->status == UptimeStatus::DOWN) {
                 $uptime->status_last_change_date = now();
             }
```

Comment:
I think this extra condition shouldn't exist. This means that the `status_last_change_date` can only be updated when the original status is different than the current status but also when the current uptime status is equal to down.